config.js

```diff
@@ -13,7 +13,6 @@
  */
 'use strict'; // eslint-disable-line strict
 require('./config/toggles'); // Loads the feature toggles
```

Comment:
Yeah I saw that too--I meant to take it out in this PR... I'll catch in the next one.